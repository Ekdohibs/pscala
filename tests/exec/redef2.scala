class A {
  def f() { print("A.f\n") }
}

class B extends A {
  override def f()  { print("B.f\n") }
}

object Main { def main(args: Array[String]) {
  var a = new A();
  var b = new B();
  a.f();
  b.f();
  var x: A = b;
  x.f()
}
           }
