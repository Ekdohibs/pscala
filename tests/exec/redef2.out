A.f
B.f
B.f
